iog-api/iog_api/models.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+
+from db import Base
+
+
+class Schemas(Base):
+    __tablename__ = 'projects'
```

Comment:
not "schemas"?

Reply:
not "schemata"?

Reply:
pushed a change to __tablename__

schemata or schema?